RequestManagementSystem/private/RequestTask.py

```diff
@@ -330,18 +328,17 @@ def __call__(self):
             monitorServer = RPCClient("WorkloadManagement/JobMonitoring", useCertificates=True)
             res = monitorServer.getJobPrimarySummary(int(self.request.JobID))
             if not res["OK"]:
-              self.log.error("RequestTask: Failed to get job %d status" % self.request.JobID)
+              self.log.error("RequestTask: Failed to get job status", "%s" % self.request.JobID)
```

Comment:
`%d` ?

Reply:
can do. But to be honest, I've never really understood what it brings, if someone could shade a light